src/components/Map/USACountyMap.js

```diff
@@ -10,9 +10,26 @@ import { useSummaries } from 'common/location_summaries';
 import { ScreenshotReady } from 'components/Screenshot';
 
 function getStateCode(stateName) {
+  // hacky, name defined slightly differently in normal reversed states file.
+  if (stateName === 'Commonwealth of the Northern Mariana Islands') {
```

Comment:
Yeah this is a good question.  The main reason I went with Northern Mariana Islands on the location page + search was because it makes the minimap pretty cramped:
![image](https://user-images.githubusercontent.com/1422280/92626970-b131c080-f298-11ea-887e-e5818f889848.png)

Reply:
I think if people are okay with the truncated `Commonwealth of t...` on the minimap, I'd prefer to just go with `Commonwealth of the Northern Mariana Islands` for consistency

Reply:
I'm not too worried about the minimap text, but I'm also not sure what the motivation is to prefer the longer name? Especially when we do have space constraints...

The wikipedia article is under "Norther Mariana Islands" (listing "Commonwealth ...") as the _official_ name..  https://en.wikipedia.org/wiki/Northern_Mariana_Islands. 🤷

Reply:
I was using Northern Mariana Islands everywhere, but the `counties-10m.json` file which defines the map and is the original list of states has it listed with Commonwealth.  I could modify that file to change the name to Northern Mariana Islands, but since we essentially downloaded it, i figured it'd be better to not modify it and shim in the state code in the javascript.  

The easier solution is probably just changing "Commonwealth of the Northern Mariana Islands" to "Northern Mariana Islands" in counties-10m.json.

Reply:
Yeah, I'm fine with modifying those files.